### PR TITLE
Multi tenancy

### PR DESCRIPTION
**Pull request by WilcoLouwerse**
https://conduction.atlassian.net/browse/MF-43

https://github.com/ConductionNL/gateway-ui/pull/346 Can be merged after this PR has been merged!

### CODE REVIEW

File: api/src/Service/SynchronizationService.php [hunk 1]
```diff
@@ -1077,26 +1050,24 @@ private function translate(array $sourceObject, bool $translateOut = false): arr
...
             ($applications = $this->entityManager->getRepository('App:Application')->findAll()
                 && !empty($applications)
                 && $application = $applications[0])
                 && $application instanceof Application
         ) {
             $objectEntity->setApplication($application);
-            $objectEntity->setOrganization($application->getOrganization());
         }
 
         return $objectEntity;
```
Comment (rjzondervan) [inline api/src/Service/SynchronizationService.php:1073]: Have we checked if this function is not used elsewhere?

File: api/src/Service/SynchronizationService.php [hunk 2]
```diff
@@ -788,21 +778,6 @@ public function findSyncByObject(ObjectEntity $objectEntity, Source $source, Ent
...
-    public function setDefaultOwner(ObjectEntity $object): ObjectEntity
-    {
-        $defaultUser = $this->entityManager->getRepository('App:User')->findOneBy(['reference' => $this->synchronizationDefault['owner']]);
-        if ($defaultUser instanceof User === false) {
-            return $object;
-        }
-
-        return $object->setOwner($defaultUser->getId()->toString());
-    }//end setDefaultOwner()
-
```
Comment (rjzondervan) [inline api/src/Service/SynchronizationService.php:805]: Same here, do we have bundles that rely on this?

**Review by rjzondervan — CHANGES_REQUESTED** [review]
Seems alright. Two questions to check.

Small request: could you rename the PR please?

File: api/src/Service/SynchronizationService.php (hunk 1 above)
Comment (WilcoLouwerse) [reply api/src/Service/SynchronizationService.php:1073]: Yes, not used in Gateway, CoreBundle, OC, KISS, XXLLNC, ZGW, DOWR bundles

File: api/src/Service/SynchronizationService.php (hunk 2 above)
Comment (WilcoLouwerse) [reply api/src/Service/SynchronizationService.php:805]: Yes, not used in Gateway, CoreBundle, OC, KISS, XXLLNC, ZGW, DOWR bundles

**Review by rjzondervan — APPROVED** [review]

**Review by bbrands02 — APPROVED** [review]

**Pull request merged**